app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -96,11 +96,16 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType: initCont
   const enableNext = indexPosition < (page?.items.length ?? 0) - 1;
 
   const determineActions = () => {
-    if (contentType === ContentTypeName.Snippet)
-      return (a: IActionModel) => a.valueType === ValueType.Boolean;
-    if (contentType === ContentTypeName.PrintContent)
-      return (a: IActionModel) =>
-        a.valueType === ValueType.Boolean && a.name !== ActionName.NonQualified;
+    switch (contentType) {
+      case ContentTypeName.Snippet:
+        return (a: IActionModel) => a.valueType === ValueType.Boolean;
+      case ContentTypeName.PrintContent:
+        return (a: IActionModel) =>
+          a.valueType === ValueType.Boolean && a.name !== ActionName.NonQualified;
+      // TODO: Determine actions for remaining content types
+      default:
+        return (a: IActionModel) => a.valueType === ValueType.Boolean;
```

Comment:
Same as top, rewrite.
```
case ContentTypeName.Snippet:
default:
   return ...
```

Reply:
For some reason ES-Lint does not like the fall through

Reply:
Still works though

Reply:
Interesting linting rule to stop fall throughs... Makes me wonder why use a switch statement if you can't use a fall through?